x/wasmplus/keeper/msg_server_integration_test.go

```diff
@@ -69,23 +111,7 @@ func TestStoreAndInstantiateContract(t *testing.T) {
 			require.NoError(t, wasmApp.AppCodec().Unmarshal(rsp.Data, &storeAndInstantiateResponse))
 
 			// check event
-			events := rsp.Events
-			assert.Equal(t, 3, len(events))
-			assert.Equal(t, "store_code", events[0].Type)
-			assert.Equal(t, 2, len(events[0].Attributes))
-			assert.Equal(t, "code_checksum", string(events[0].Attributes[0].Key))
-			assert.Equal(t, "code_id", string(events[0].Attributes[1].Key))
-			assert.Equal(t, "1", string(events[0].Attributes[1].Value))
-			assert.Equal(t, "message", events[1].Type)
-			assert.Equal(t, 2, len(events[1].Attributes))
-			assert.Equal(t, "module", string(events[1].Attributes[0].Key))
-			assert.Equal(t, "wasm", string(events[1].Attributes[0].Value))
-			assert.Equal(t, "sender", string(events[1].Attributes[1].Key))
-			assert.Equal(t, "instantiate", events[2].Type)
-			assert.Equal(t, "_contract_address", string(events[2].Attributes[0].Key))
-			assert.Equal(t, storeAndInstantiateResponse.Address, string(events[2].Attributes[0].Value))
-			assert.Equal(t, "code_id", string(events[2].Attributes[1].Key))
-			assert.Equal(t, "1", string(events[2].Attributes[1].Value))
+			assert.True(t, reflect.DeepEqual(spec.events, rsp.Events))
```

Comment:
Please check https://github.com/Finschia/wasmd/pull/76#discussion_r1298307663

Reply:
Thanks. I fixed it as pointed out.